resources/views/components/molecules/_m-listing----stories-listing.blade.php

```diff
@@ -37,13 +37,36 @@
                 @endcomponent
                 <br>
                 @if ($isFeatured)
+                    @if (isset($item->authors) && (count($item->authors) > 0) || $item->author_display)
```

Comment:
You can use `empty($item->authors)` instead of `isset($item->authors) && (count($item->authors) > 0)`. For reference see
https://www.php.net/manual/en/function.empty.php

Reply:
@zachgarwood I tried making empty work here but since it's evaluating a collection at a top level there are properties in the collection that will always eval !empty as true.